packages/create-neutrino-project/commands/init/index.js

```diff
@@ -0,0 +1,187 @@
+const { ensureDirSync, readJsonSync, writeJsonSync } = require('fs-extra');
+const path = require('path');
+const chalk = require('chalk');
+const stringify = require('javascript-stringify');
+const Generator = require('yeoman-generator');
+const questions = require('./questions');
+const {
+  PROJECTS,
+  LIBRARIES,
+  LINTERS,
+  isYarn,
+  isObjectEmpty
+} = require('./utils');
+
+module.exports = class Project extends Generator {
+  _makeRcFile(data) {
+    const config = { use: [] };
+
+    config.use.push(data.projectType);
+
+    switch (data.linter) {
+      case LINTERS.AIRBNB:
+        config.use.push(
+          data.projectType === PROJECTS.REACT ? LIBRARIES.NEUTRINO_PRESET_AIRBNB : LIBRARIES.NEUTRINO_PRESET_AIRBNB_BASE
+        );
+        break;
+      case LINTERS.STANDARDJS:
+        config.use.push(LIBRARIES.NEUTRINO_MIDDLEWARE_STANDARDJS);
+        break;
+      default:
+        break;
+    }
+
+    if (data.testRunner) {
+      config.use.push(data.testRunner);
+    }
+
+    return `module.exports = ${stringify(config, null, 2)};\n`;
+  }
+
+  _getDependencies() {
+    const devDependencies = [];
+    const dependencies = [];
+
+    switch (this.data.projectType) {
+      case PROJECTS.REACT:
+        dependencies.push(LIBRARIES.REACT, LIBRARIES.REACT_DOM, LIBRARIES.REACT_HOT_LOADER);
+        devDependencies.push(LIBRARIES.NEUTRINO_PRESET_REACT);
+        break;
+      case PROJECTS.REACT_COMPONENTS:
+        dependencies.push(
```

Comment:
For a react-components project, I think these are all dev dependencies.

Reply:
@eliperelman How come? the [installation](https://github.com/eliperelman/neutrino-preset-react-components#installation) of neutrino-preset-react-components suggests that `react`, `react-dom`, `react-addons-css-transition-group` are added without the dev flag. Is that a mistake in the README? Hm, maybe I should only change `LIBRARIES.NEUTRINO_PRESET_REACT_COMPONENTS` to be a dev dependency?

Reply:
Yeah, I think that's a mistake in the readme. For example, if you look at the dependencies of react-lazylog and react-fout-stager:

https://github.com/mozilla-rpweb/react-lazylog/blob/6a10676edacedcf0fb84e6165ef75e33aa7fbc80/package.json#L27-L34

https://github.com/eliperelman/react-fout-stager/blob/624befe3fc7c7f019164fc71d975de2557e396da/package.json#L17-L24

You'll notice everything is in devDependencies that aren't direct for the component.